### PR TITLE
Fixed issue with multiple pfdhcplisteners not stopping properly

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Fixed issue with multiple pfdhcplisteners not stopping properly
## Impacts

bin/pfcmd service
## Delete branch after merge

NO

### CODE REVIEW

**Comment by julsemaan** [comment]
News file!

**Comment by julsemaan** [comment]
Tested and it doesn't spawn duplicate processes anymore.

**Comment by jrouzierinverse** [comment]
Will update news file after #309 pull request is merged

**Comment by jrouzierinverse** [comment]
Done

File: NEWS.asciidoc [hunk 1]
```diff
@@ -57,6 +57,7 @@ Bug Fixes
 * Fixed URI based portal profiles
 * Fixed XSS vulnerabilities in the portal
 * Refresh node page after updating a node
+* Fixed issued which caused multiple pfdhcplistener to be spawned
```
Comment (julsemaan) [inline NEWS.asciidoc]: *Fixed issue
*multiple pfdhcplistener processes to be spawned

**Pull request merged**